Sample sites/jquery/pages/update-profile-xmpl-and-js-syntax.html

```diff
@@ -78,6 +78,12 @@ <h3 class="title">Profile</h3>
               <input class="input__element" type="text" size="30" xmp-write-ador="xmp.r['Feedback']" />
             </div>
           </div>
+          <div class='form__section form__section_sm'>
+            <div class='form__group'>
+              <input type="checkbox" xmp-write-ador="xmp.r['Gift']" 
```

Comment:
We don't have ador "Gift" in the project
You can use Feedback ador or ask Keren how to add new one